CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 -
 
+# v6.5.0
+
+- [added] `FirebaseOptions.Builder` class now provides a
```

Comment:
FWIW we use "feature" in our other SDK changelogs I think (and in the actual public release notes: https://firebase.google.com/support/release-notes/admin/node)

Reply:
Yes, for some reason this file has been set up with a slightly different terminology. I plan to make it consistent with our published release note files in the future.